Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBGridFSBucket-find.txt

MongoDB\\GridFS\\Bucket::find()
===============================

.. meta::

>    :description: Discover how to use `MongoDB\GridFS\Bucket::find()` to query documents in a GridFS bucket's files collection with various filter and option parameters.

Reviewer: ```suggestion
   :description: Query documents in a GridFS bucket's files collection with the MongoDB PHP Library, with various filter and option parameters.
```
